addons/website/views/website_templates.xml

```diff
@@ -566,44 +566,16 @@
         <!-- Font options -->
         <content id="theme_customize_content_fonts" string="Fonts" title="Choose your fonts">
             <list string="Title">
-                <selection id="theme_customize_content_fonts_title">
```

Comment:
Nice but now you must create a theme branch to adapt the theme extension (it should already have been done for step 1 in fact :)). At the end, we won't need theme extension thanks to step 2, so in the theme repo, this can be just 1 commit which removes the necessary stuff :)